web/ui/two-steps-page.svelte

```diff
@@ -22,19 +23,33 @@
     })
   }
 
+  function toggleStepsOnEscape(event: KeyboardEvent): void {
+    if (event.key === 'Escape' && document.activeElement?.tagName === 'BODY') {
+      toggleSteps()
```

Comment:
Do you have a reason why on `Esc` on the first step we should move to the second?

Reply:
No, I just thought I should implement switching between sides using the Escape key.

Reply:
Let’s use `showFirstStep()` here.

I'm afraid that we will have unexpected behavior of switching 1→2 on `Esc` key.